nsxt/resource_nsxt_lb_pool.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 var poolAlgTypeValues = []string{"ROUND_ROBIN", "WEIGHTED_ROUND_ROBIN", "LEAST_CONNECTION", "WEIGHTED_LEAST_CONNECTION", "IP_HASH"}
-var poolSnatTranslationTypeValues = []string{"LbSnatAutoMap", "Transparent"}
+var poolSnatTranslationTypeValues = []string{"LbSnatAutoMap", "LbSnatIpPool", "Transparent"}
```

Comment:
I think enumeration that we generally exposed are uppercase. Lets change this to "AUTO_MAP", "IP_POOL", "TRANSPARENT" and translate?

Reply:
Thanks. Fixed